_episodes_rmd/04-cond.Rmd

```diff
@@ -152,7 +152,7 @@ sign(0)
 sign(2/3)
 ```
 
-Note that when combining `else` and `if` in an `else if` statement (similar to `elif` in Python), the `if` portion still requires a direct input condition.  This is never the case for the `else` statement alone, which is only executed if all other conditions go unsatisfied.
+Note that when combining `else` and `if` in an `else if` statement, the `if` portion still requires a direct input condition.  This is never the case for the `else` statement alone, which is only executed if all other conditions go unsatisfied.
```

Comment:
doesn't seem helpful to make analogy to Python (most students will be unfamiliar)